cli/src/commands/debug/init_local.rs

```diff
@@ -25,10 +25,15 @@ use crate::command_error::user_error_with_message;
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Create a new repo in the given directory
+/// Create a new repo in the given directory using the proof-of-concept native
+/// backend
 ///
-/// If the given directory does not exist, it will be created. If no directory
-/// is given, the current directory is used.
+/// The proof-of-concept backend is called "local" because it does not support
+/// cloning, pushing, or fetching repositories or commits.
```

Comment:
It's called "local" because it stores commits locally (unlike the Google backend), but it doesn't really matter since almost no one will read this anyway :)

Reply:
That'd make it confusable with the `git` backend, so clearly it's *now* named the "local" backend for another reason. 😛